app/konnect/runtime-manager/runtime-instances/index.md

```diff
@@ -1,24 +1,44 @@
 ---
 title: About Runtime Instances
 content_type: explanation
+disable_image_expand: true
 ---
 
-
 A runtime instance is a single self-managed instance of {{site.base_gateway}} that functions as a data plane. In {{site.konnect_saas}}, runtime instances are part of [runtime groups](/konnect/runtime-manager/runtime-groups) and can be used to serve traffic to the runtime group. Runtime groups manage and store configuration in {{site.konnect_saas}}, and runtime instances are configured according to the configuration distributed by the runtime group. 
 
-
 {{site.konnect_short_name}} provides runtime instance installation scripts for various platforms. These runtime instances are configured to run in your {{site.konnect_short_name}} environment.
 You can access these scripts from the {{site.konnect_short_name}} **Runtime Manager**. 
 
+## Supported installation options
+
+{:.note}
+> **Note:** Kong does not host runtime instances. You must install and host your own.
+
 {% include install.html config=site.data.tables.install_options_konnect header='no-header' %}
 
+## Communicating through a forward proxy 
+
+In situations where forward proxies are non-transparent, you can still connect the {{site.base_gateway}} data plane with the {{site.konnect_saas}} control plane. 
+To do this, you need to configure each {{site.base_gateway}} runtime instance to authenticate with the proxy server and allow traffic through.
+For more information, see [Control Plane and Data Plane Communication through a Forward Proxy](/gateway/latest/production/networking/cp-dp-proxy/) in the {{site.base_gateway}} documentation.
+
+## Running on AWS
```

Comment:
This information should live in the UI, or in a FAQ style topic in a section titled: "What resources are created when I deploy on AWS". This is info you should probably know before you hit the deploy to AWS button in the UI

Alone here makes it seem like we forgot any info about Google, or Azure. If we list what resources are created in AWS, why wouldn't we list Azure or Google Cloud?